### ISSUE TITLE
Question editor page: ARIA IDs aren't unique

### ISSUE THREAD

**Issue by neel-a-mehta** [civiform/civiform#1146]
## Problem

As a CiviForm admin, I went to edit a particular question and ran Lighthouse on it. I got this problem:

![image](https://user-images.githubusercontent.com/77128518/118336554-e6557900-b4df-11eb-9f5e-e89d649c9c6a.png)

> The value of an ARIA ID must be unique to prevent other instances from being overlooked by assistive technologies. Learn
more.
> Failing Elements
> input#options[].border-gray-300.bg-gray-50.block.border.p-2.w-full

I think I was editing a checkbox question, but I'm not entire sure. 

## Solution 

In general, when rendering repeated elements like a list of checkbox items, we should make sure that each element gets a unique ARIA label for accessibility's sake.

**Comment by jeanniefu** [comment]
Note: we are focusing on applicant accessibility for now, so this bug is low priority as it only affects the admin experience.

**Comment by bion** [comment]
We do not have an a11y for admin-facing pages
